### PR TITLE
[wip] Sec tab

### PR DESCRIPTION
**Pull request by ghost**
TODO:
- [ ] handle patching and creating by external_vulnerability_id
- [ ] regenerate migrations
- [ ] better html (now br)
- [ ] docstings

### CODE REVIEW

File: src/ralph/admin/sitetrees.py [hunk 1]
```diff
@@ -135,6 +135,24 @@ def section(section_name, app, model):
...
+            url_as_pattern=False,
+            perms_mode_all=False,
+            children=[
+                section(
+                    section_name=_('Security Scans'),
+                    app='security',
+                    model='SecurityScan'
+                ),
+                section(
+                    section_name=_('Vulnerability'),
```
Comment (mkurek) [inline src/ralph/admin/sitetrees.py]: Vulnerabilities?

File: src/ralph/data_center/templates/data_center/datacenterasset/security_info.html [hunk 2]
```diff
@@ -1,63 +1,46 @@
...
-      <tbody>
-        {% for item in data_classification %}
-          <tr>
-            <td>{{ item.label }}</td>
-            <td>{{ item.value }}</td>
-          </tr>
-        {% endfor %}
-      </tbody>
+
+  <h1>Security scan</h1>
```
Comment (mkurek) [inline src/ralph/data_center/templates/data_center/datacenterasset/security_info.html]: add trans (everywhere :))

File: src/ralph/security/models.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    TimeStampMixin,
+    TaggableMixin,
+    models.Model,
+):
+    last_scan_date = models.DateTimeField()
+    scan_status = models.PositiveIntegerField(choices=ScanStatus())
+    next_scan_date = models.DateTimeField()
+    details_url = models.URLField(max_length=255, blank=True)
+    rescan_url = models.URLField(blank=True, verbose_name='Rescan url')
+    asset = models.OneToOneField(BaseObject, primary_key=True)
```
Comment (mkurek) [inline src/ralph/security/models.py]: to think about if it should be OneToOneField or ForeignKey (ex. what if one asset has multiple ip addresses)?

**Comment by vi4m** [comment]
Please create new branch on allegro github called "feature/security"  and make pull request there.

**Comment by mkurek** [comment]
I'm closing it for now - we'll reopen it when we'll merge it into ng branch; check #2002  now

**Pull request closed without merging**